src/index.html

```diff
@@ -10,13 +10,74 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYOlogo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-links">
+          <li class="link">
+            <a
+              href="index.html"
+              class="is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="link">
+            <a href="index.html">samsung</a>
+          </li>
+          <li class="link">
+            <a href="index.html">smartphones</a>
```

Comment:
The `href` attribute is set to 'index.html'. If this link is intended to navigate to a different section or page, please ensure the `href` attribute is correctly set to the respective URL.